cvat/apps/engine/views.py

```diff
@@ -210,15 +216,11 @@ def get_serializer(self, *args, **kwargs):
     def about(request):
         from cvat import __version__ as cvat_version
         about = {
-            "name": "Computer Vision Annotation Tool",
+            "name": settings.ABOUT_INFO["name"],
+            "title": settings.ABOUT_INFO["title"],
+            "description": settings.ABOUT_INFO["description"],
             "version": cvat_version,
-            "description": "CVAT is completely re-designed and re-implemented " +
-                "version of Video Annotation Tool from Irvine, California " +
-                "tool. It is free, online, interactive video and image annotation " +
-                "tool for computer vision. It is being used by our team to " +
-                "annotate million of objects with different properties. Many UI " +
-                "and UX decisions are based on feedbacks from professional data " +
-                "annotation team."
+            "logo": get_logo(),
```

Comment:
Do we really need to include the whole logo into the API response?

I think it would be more sensible to include the URL of the logo, and stick the actual logo in the `static` directory. Then:

1. The API response won't be as bloated.
2. The usual HTTP caching mechanisms will apply, so the clients won't fetch the logo more often than necessary.
3. It'll be more efficient if the logo is in a non-text format (e.g. PNG), since no encoding will be necessary.